source/code-snippets/usage-examples/bulkWrite-example.js

```diff
@@ -1,15 +1,11 @@
 const { MongoClient } = require("mongodb");
 const fs = require("fs");
 
-// Replace the following with your MongoDB deployment's connection
-// string.
+// Replace the following string with your MongoDB deployment's connection string.
 const uri =
-  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority&useUnifiedTopology=true";
 
-const client = new MongoClient(uri, {
-  useNewUrlParser: true,
-  useUnifiedTopology: true
```

Comment:
Considering that most of our users will (as our comment suggests) *replace* the URI above, I think adding options to it likely won't have the intended result. If the new url parser is now used by default, I'd suggest leaving the client unified topology option in place. It also helps demonstrate how to pass options to the client, which has a lot of value for a usage example IMO.

Reply:
Fixed

Reply:
Fixed